.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -38,23 +38,24 @@ jobs:
           URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
           UPLIFT_PR_REGEX='uplift\s+(?:of)?:?\s+\K(#\d+)'
           RESOLVE_ISSUE_REGEX="(?:resolv|fix|clos)(?:e|es|ed)?:?\s+\K(${URL_REGEX})"
+          verlte() { printf '%s\n%s' "$1" "$2" | sort -C -V; }
 
-          uplift_pr_body=$(gh pr view "$PR_NUMBER" -R "$REPO_NAME" --json body --jq '.body')
+          uplift_pr_body=$(gh pr view "${PR_NUMBER}" -R "${REPO_NAME}" --json body --jq ".body")
 
           issues=$(echo "${uplift_pr_body}" | grep -ioP "${RESOLVE_ISSUE_REGEX}" || :)
           if [[ -z ${issues} ]]; then
               echo "Resolved issues not directly found. Following PRs to determine them..."
               prs=$(echo "${uplift_pr_body}" | grep -ioP "${UPLIFT_PR_REGEX}")
-              issues=$(xargs -n 1 -I{} gh pr view {} -R brave/brave-core --json body --jq '.body' < <(echo "${prs}") | grep -ioP "${RESOLVE_ISSUE_REGEX}")
+              issues=$(xargs -n 1 -I{} gh pr view {} -R brave/brave-core --json body --jq ".body" < <(echo "${prs}") | grep -ioP "${RESOLVE_ISSUE_REGEX}")
```

Comment:
It's safe to just do `--jq .body`

Reply:
Also, not a fan of reusing variables (assigning values more than once) . It's confusing. But feel free to keep it, if you prefer.